src/phpdoc/type_parser.go

```diff
@@ -429,6 +464,15 @@ func (p *TypeParser) nextByte() byte {
 	return 0
 }
 
+func (p *TypeParser) lookAhead() byte {
```

Comment:
We don't need this method. Please maintain an index and use `peekAt`. The less scan/lex methods we have, the better.